sections/07_conclusions.md

```diff
@@ -52,6 +52,23 @@ addition, the stochasticity and complexity of biological systems implies that
 for some problems, for instance, predicting gene regulation in disease, perfect
 accuracy will be unattainable.
 
+We are witnessing deep learning models achieving human-level performance across
+a number of biomedical domains, and yet we do not believe that biologists and
+clinicians will be out of a job anytime soon. While deep learning and other
+advanced algorithms will soon be (or already are) better than scientists at
+specific tasks, in many cases they would still be unable to fully grasp the
+bigger picture. Deep neural networks are also prone to mistakes that humans are
```

Comment:
You've been following the discussion in #384, and I agree that this is a more solid way to make this point.  We can replace "such as incorrectly classifying negative images [@arxiv:1703.06857]" with something like "such as susceptibility to adversarial examples [a couple references from #384]".

In the next sentence, you can acknowledge defensive techniques and then continue with the point you want to make: "Despite progress in addressing specific limitations such as adversarial inputs [a couple references from #384 or Section 2.3 of the paper I linked], until true and reliable..."